app/communication/models/banner.py

```diff
@@ -0,0 +1,63 @@
+import uuid
+
+from django.db import models
+
+from app.common.enums import AdminGroup
+from app.common.permissions import BasePermissionModel
+from app.communication.exceptions import (
+    AnotherVisibleBannerError,
+    DatesMixedError,
+)
+from app.communication.mixins import APIBannerErrorsMixin
+from app.util.models import BaseModel, OptionalImage
+from app.util.utils import now, tomorrow
+
+
+class Banner(BaseModel, OptionalImage, BasePermissionModel, APIBannerErrorsMixin):
+    write_access = AdminGroup.admin()
+    read_access = AdminGroup.admin()
+
+    id = models.UUIDField(
+        auto_created=True,
+        primary_key=True,
+        default=uuid.uuid4,
+        serialize=False,
+    )
+    title = models.CharField(max_length=100)
+    description = models.TextField()
+    visible_from = models.DateTimeField(default=now)
+    visible_until = models.DateTimeField(default=tomorrow)
+    url = models.URLField(max_length=600, blank=True, null=True)
+
+    def __str__(self):
+        return f"{self.title} {'- is visible' if self.is_visible else ''} - {self.description}"
+
+    def save(self, *args, **kwargs):
+        if self.is_visible and self.exists_other_visible_banners:
+            raise AnotherVisibleBannerError(
+                "Det finnes allerede et banner som er synlig"
+            )
```

Comment:
Denne sjekken vil vel kun funke hvis man lager et banner som skal være synlig akkkurat nå. Kun da vil `self.is_visible` være True. Tror egentlig vi bare kan fjerne `is_visible`-propertien siden den ikke brukes for å filtrere i viewet uansett

Reply:
Ja det er sant. 
Endrer det til dette istedet. Med tanke på navnet tenker jeg det eksisterende navnet passer bedre enn `exists_conflicting_banner `siden det er mer beskrivende for visible. Men bra forslag. Jeg kan se om det finnes et enda bedre navn.
```
    @property
    def exists_other_visible_banners(self):
        return (Banner.objects
        .filter(visible_from__lte=self.visible_until, visible_until__gte=self.visible_from)
        .exclude(id=self.id)
        .exists())
```
Denne sjekken virker ikke om `visible_from `er etter `visible_until`, men vi har en sjekk på det etter uansett.

Reply:
`exists_overlapping_banners` høres enda bedre ut. Spot on synes jeg